Siri/L4/Level4.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Level4 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> arrList = new ArrayList<Task>();
```

Comment:
perhaps changing the variable name to be more meaningful (e.g. taskList, tasks)?